src/main/java/com/bravo/user/exception/PaymentNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.bravo.user.exception;
+
+import java.text.MessageFormat;
+
+public class PaymentNotFoundException extends RuntimeException {
+
+    private static final String MESSAGE = "Payment not found for user with id {0}";
+
+    public PaymentNotFoundException(String userId) {
+        super(MessageFormat.format(MESSAGE, userId));
```

Comment:
1. Why did you use `MessageFormat.format` instead of `String.format`? 
2. Do you think this `MESSAGE` variable is really necessary as it's only used in 1 place?

Reply:
@wheelerswebservices regarding first: I don't think this is really matters
second: this exception is only related to Payment not found flow, so there is not reason to extract this message somewhere else.